sql/index/pilosa/lookup.go

```diff
@@ -81,6 +79,8 @@ func (l *indexLookup) bitmapQuery() (*pilosa.PQLBitmapQuery, error) {
 			b, e = il.bitmapQuery()
 		case *descendLookup:
 			b, e = il.bitmapQuery()
+		case *negateLookup:
```

Comment:
shall we create a common pilosa Lookup interface with `BitmapQuery()` method? WDYT?

Reply:
yup, sounds good